src/tiled/tileanimationeditor.cpp

```diff
@@ -257,6 +259,13 @@ const QVector<Frame> &FrameListModel::frames() const
 {
     return mFrames;
 }
+void FrameListModel::setDefaultFrameTime(int duration)
+{
+    DEFAULT_DURATION=duration;
+    for(Frame &F:mFrames){
```

Comment:
Coding style: `for (Frame &F:mFrames) {`

And since this is a single line statement, you can get rid of the braces.

Reply:
Thanks, Change has been made.